src/services/syncFeedPerUser.js

```diff
@@ -189,8 +189,8 @@ const resetAndSyncFeed = async (req, res) => {
       await syncMainFeedFollowing(userId);
     } else if (feed === 'main_feed_f2') {
       await syncMainFeedF2(userId);
-    } else if (feed === 'main_feed_broad') {
-      await syncMainFeedBroad(userId);
+      // } else if (feed === 'main_feed_broad') {
+      //   await syncMainFeedBroad(userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `syncMainFeedBroad` has been commented out. If this functionality is no longer required, it's better to remove these lines completely to avoid confusion.

```diff
- // } else if (feed === 'main_feed_broad') {
- //   await syncMainFeedBroad(userId);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->